packages/flutter/lib/src/material/outline_button.dart

```diff
@@ -183,7 +185,7 @@ class OutlineButton extends MaterialButton {
 
 // The type of of OutlineButtons created with [OutlineButton.icon].
 //
-// This class only exists to give RaisedButtons created with [RaisedButton.icon]
+// This class only exists to give OutlineButtons created with [OutlineButton.icon]
```

Comment:
Since this is a regular comment, the [links] don't do anything...
```suggestion
// This class only exists to give OutlineButtons created with OutlineButton.icon
```